src/index.html

```diff
@@ -22,6 +22,421 @@
+                <a
+                  href="Samsung"
+                  class="nav_link"
+                >
+                  Samsung
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Smartphones"
+                  class="nav_link"
+                >
+                  Smartphones
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Laptops & Computers"
+                  class="nav_link"
+                  data-qa="nav-hover"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Gadgets"
+                  class="nav_link"
+                >
+                  Gadgets
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Tablets"
+                  class="nav_link"
+                >
+                  Tablets
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Photo"
+                  class="nav_link"
+                >
+                  Photo
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="Video"
+                  class="nav_link"
+                >
+                  Video
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <main class="main">
+        <div class="product-cards-container">
+          <div
+            class="product-card"
+            data-qa="card"
+          >
+            <img
+              src="images/imac.jpeg"
+              alt="imac"
+              class="product-card__img"
+            />
+
+            <p class="product-card__name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+
+            <p class="product-card__code">Product code: 195434</p>
+
+            <div class="product-card__container">
+              <div class="stars stars--4">
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+              </div>
+              <p class="product-card__reviews">Reviews: 5</p>
+            </div>
+
+            <div class="product-card__prices">
+              <p class="product-card__price">Price:</p>
+              <p class="product-card__value">$2,199</p>
+            </div>
+
+            <button
+              class="product-card__buy"
+              data-qa="card-hover"
+            >
+              BUY
+            </button>
+          </div>
+        </div>
+        <div class="product-cards-container">
```

Comment:
Add the blank line between the cards
```suggestion
        </div>
        
        <div class="product-cards-container">
```